### DISCUSSION TITLE
stupid watermark

### DISCUSSION THREAD

**Discussion by Kenocrystal** [OhMyGuus/BetterCrewLink#282] [Q&A] [answered]
any way to remove the dumb, annoying watermark that shows up over your game?

**Comment by OhMyGuus** [comment, marked as answer]
if u mean the annoying mod icon by the settings icon  then nope since we are required by innersloth to show it. We also only enable that icon the style lf the icon and the position is all innersloth work. (I will disable it tho in the next update when ToR is loaded because it shows 2 watermarks) 

If u mean the text bettercrewlink.app under the ping then u might have to live with that as well, but I might try to disable it in a meeting if possible if people find it too anoying.
